logstash-core/lib/logstash/api/lib/app.rb

```diff
@@ -25,6 +25,19 @@ def initialize(app=nil)
       @factory = CommandFactory.new(settings.service)
     end
 
+    get "/_directory" do
```

Comment:
The most I saw is having proper documentation in our case we've
https://www.elastic.co/guide/en/logstash/5.0/monitoring.html, or custom
solutions, any pointer would be helpful. Basic idea of resources like this
is enable easy discoverly, as we know no one really looks at documentation.

- purbon

On Thu, May 12, 2016 at 2:43 PM Pier-Hugues Pellerin <
notifications@github.com> wrote:

> In logstash-core/lib/logstash/api/lib/app.rb
> <https://github.com/elastic/logstash/pull/5069#discussion_r63012799>:
>
> > @@ -25,6 +25,19 @@ def initialize(app=nil)
> >        @factory = CommandFactory.new(settings.service)
> >      end
> >
> > +    get "/_directory" do
>
> Is there any prior art for the name of this route? Maybe if we look at
> hypermedia type of api?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/logstash/pull/5069/files/285c2e225d1f2b7b2bf8faaf969ffb443d7ff23f#r63012799>
>